src/pages/tutor-profile/TutorProfile.jsx

```diff
@@ -7,23 +7,32 @@ import VideoPresentation from '~/containers/tutor-profile/video-presentation/Vid
 import CommentsWithRatingBlock from '~/containers/tutor-profile/comments-with-rating-block/CommentsWithRaitngBlock'
 
 import { responseMock } from '~/pages/tutor-profile/constants'
+import { parseJwt } from '~/utils/helper-functions'
+import useUserInfo from '~/hooks/use-user-info'
+import Loader from '~/components/loader/Loader'
 
 const TutorProfile = () => {
   const { user } = responseMock
   const { averageRating, reviews, totalReviews } = user.reviewStats || {}
 
+  const { id, role } = parseJwt(localStorage.getItem('s2s'))
+
+  const { loading, response: userData } = useUserInfo({ id, role })
+
+  if (loading) {
+    return <Loader pageLoad size={70} />
+  }
+
   return (
     <Container sx={{ flex: 1, pb: '100px' }}>
-      <ProfileInfo />
+      <ProfileInfo userData={userData} />
       <CompleteProfileBlock data={{}} profileItems={profileItems} />
```

Comment:
Why you don't pass the data to CompleteProfileBlock?
If you pass data to CompleteProfileBlock you will see a progress bar.

Reply:
Fixed. I presumed CompleteProfileBlock isn't functioning

Reply:
Fixed.